src/objects/Lotto.spec.js

```diff
@@ -0,0 +1,94 @@
+import { ERROR_MESSAGE, LOTTO_RANK_MAP } from '../lib/constants.js';
+import { Lotto } from './index.js';
+
+describe('Lotto', () => {
+  describe('예외 케이스', () => {
+    test('로또 번호의 개수가 6개가 넘어가는 경우 예외가 발생한다.', () => {
+      expect(() => {
+        new Lotto([1, 2, 3, 4, 5, 6, 7]);
+      }).toThrow(ERROR_MESSAGE.NOT_SIX);
+    });
+
+    test('로또 번호에 중복된 숫자가 있는 경우 예외가 발생한다.', () => {
+      expect(() => {
+        new Lotto([1, 2, 3, 4, 5, 5]);
+      }).toThrow(ERROR_MESSAGE.NOT_UNIQUE);
+    });
+  });
+
+  describe('정상 케이스', () => {
+    let lotto;
+
+    beforeEach(() => {
+      lotto = new Lotto([4, 2, 3, 6, 1, 5]);
+    });
+
+    describe('#sortByAscending', () => {
+      test('로또 번호가 오름차순으로 정렬된다.', () => {
+        expect(lotto.numbers).toEqual([1, 2, 3, 4, 5, 6]);
+      });
+    });
+
+    describe('getRank', () => {
+      test('로또 번호가 0개에서 2개 사이 당첨 될 경우 undefined를 반환한다.', () => {
+        expect(lotto.getRank([11, 12, 13, 14, 15, 16], 20)).toBeNull();
+        expect(lotto.getRank([1, 11, 12, 13, 14, 15], 20)).toBeNull();
+        expect(lotto.getRank([1, 2, 11, 12, 13, 14], 20)).toBeNull();
+      });
```

Comment:
이 경우 `undefined`가 아니라 `null`을 반환하는 것이니 테스트 케이스 이름을 수정하시면 어떨까요?
```suggestion
      test('로또 번호가 0개에서 2개 사이 당첨 될 경우 null을 반환한다.', () => {
        expect(lotto.getRank([11, 12, 13, 14, 15, 16], 20)).toBeNull();
        expect(lotto.getRank([1, 11, 12, 13, 14, 15], 20)).toBeNull();
        expect(lotto.getRank([1, 2, 11, 12, 13, 14], 20)).toBeNull();
      });
```

Reply:
그 방식이 맞는 거 같습니다. 감사합니다 !!